torch/nn/parallel/data_parallel.py

```diff
@@ -168,7 +172,7 @@ def __init__(
         if len(self.device_ids) == 1:
             self.module.to(self.src_device_obj)
 
-    def forward(self, *inputs: Any, **kwargs: Any) -> Any:
+    def forward(self, *inputs: P.args, **kwargs: P.kwargs) -> R:
```

Comment:
So the return type change here is good, however ParamSpec unfortunately wont do anything here because it's not bound to a callable, sadly. :(